### PR TITLE
Update `SoftmaxAggregation` and `PowerMeanAggregation` doc strings

### PR DESCRIPTION
**Pull request by lightaime**
This PR updates the docs strings of `SoftmaxAggregation` and `PowerMeanAggregation`.

Addresses https://github.com/pyg-team/pytorch_geometric/issues/4712.

### CODE REVIEW

**Review by Padarn — APPROVED** [review]

File: torch_geometric/nn/aggr/basic.py [hunk 1]
```diff
@@ -130,6 +129,24 @@ def __repr__(self) -> str:
 
 
 class PowerMeanAggregation(Aggregation):
+    r"""The powermean aggregation operator based on a power term, as
+    described in the `"DeeperGCN: All You Need to Train Deeper GCNs"
+    <https://arxiv.org/abs/2006.07739>`_ paper
+
+    .. math::
+        \mathrm{PowerMean}(\mathcal{X}|p) = (\frac{1}{|\mathcal{X}|}
```
Comment (rusty1s) [inline torch_geometric/nn/aggr/basic.py:137]:
```suggestion
        \mathrm{PowerMean}(\mathcal{X}|p) = \left(\frac{1}{|\mathcal{X}|}
```

File: torch_geometric/nn/aggr/basic.py [hunk 2]
```diff
@@ -130,6 +129,24 @@ def __repr__(self) -> str:
...
+        \sum_{\mathbf{x}_i\in\mathcal{X}}\mathbf{x}_i^{p})^{1/p},
```
Comment (rusty1s) [inline torch_geometric/nn/aggr/basic.py:138]:
```suggestion
        \sum_{\mathbf{x}_i\in\mathcal{X}}\mathbf{x}_i^{p}\right)^{1/p},
```

File: torch_geometric/nn/aggr/basic.py (hunk 1 above)
Comment (rusty1s) [inline torch_geometric/nn/aggr/basic.py:137]:
```suggestion
        \mathrm{powermean}(\mathcal{X}|p) = (\frac{1}{|\mathcal{X}|}
```

File: torch_geometric/nn/aggr/basic.py [hunk 3]
```diff
@@ -72,10 +72,9 @@ class SoftmaxAggregation(Aggregation):
     <https://arxiv.org/abs/2006.07739>`_ paper
 
     .. math::
-        \mathrm{softmax}(\{ \mathbf{x}_i : i \in |\mathcal{X}| \}, t)
-        = \sum_{i \in |\mathcal{X}|} \frac{ \exp( t \cdot \mathbf{x}_i)}
-        {\sum_{j \in |\mathcal{X}|} \exp(t \cdot \mathbf{x}_j)}
-        \cdot \mathbf{x}_{i},
+        \mathrm{Softmax}(\mathcal{X}|t) = \sum_{\mathbf{x}_i\in\mathcal{X}}
```
Comment (rusty1s) [inline torch_geometric/nn/aggr/basic.py:75]:
```suggestion
        \mathrm{softmax}(\mathcal{X}|t) = \sum_{\mathbf{x}_i\in\mathcal{X}}
```

**Review by rusty1s — APPROVED** [review]
Thanks! This is indeed cleaner :)

**Pull request merged**